### DISCUSSION TITLE
How well does sd-scripts work with ROCm and AMD?

### DISCUSSION THREAD

**Discussion by nonetrix** [kohya-ss/sd-scripts#775] [Q&A] [answered]
I am thinking about getting a AMD card, how has it been to use sd-scripts with ROCm on Linux for everyone personally? Is the experience good besides the extra setup needed? I also want to train SDXL Lora which is the main reason I want to get a new GPU is for the extra VRAM and AMD gives you a lot more for less

**Comment by jubiloso** [comment, marked as answer]
TLDR: on Linux works.

Hi, I have trained several LORAs with an AMD Radeon 6700 XT 12GB. With SD 1.5 it's easy, in ~1h 15m you can train a face. For SDXL 1.0 at 1024x1024 it's another story, I have spent more than 20 hours training a character and still doesn't get the features of the face ok. My  configuration must not be optimal for SDXL with my card, but still, training SDXL is costly. At least works.

**Reply by nonetrix** [reply]
You got SDXL to even train at all? I have been getting OOMs with setups people say work on NVIDIA cards with like 12GBs of VRAM when I have 16GBs, it's ridiculous. I've tried CPU training even because I'm so desperate, and it manages to somehow fill at my 64GBs of RAM. I might end up going back to NVIDIA, my experience has been a nightmare but problem is NVIDIAs prices are insane

**Reply by nonetrix** [reply]
I managed to get further with Anima which has same issue by asking Gemini-cli to make tweaks to code until it works, I am on iteration 591/1080 no OOMs so far. If everything goes well I'll upload my fork, probably won't make a PR since the changes probably aren't ideal and I don't understand them. But insanely impressed I've been able to not do a single iteration to getting this far just by prompting AI to fix it

![20260217_034651](https://github.com/user-attachments/assets/f7c150bf-c07d-4742-9543-10d5b76ddd43)

Edit: training completed successfully, I haven't tested the model yet though

**Reply by jubiloso** [reply]
that was more than 2 years ago.
